src/sonic-yang-models/tests/yang_model_tests/tests_config/sflow.json

```diff
@@ -59,6 +59,69 @@
         }
     },
 
+    "SFLOW_TEST_WITH_COLLECTOR_DEFAULT_VRF": {
+        "sonic-sflow:sonic-sflow": {
+            "sonic-sflow:SFLOW_COLLECTOR": {
+                "SFLOW_COLLECTOR_LIST": [
+                    {
+                        "name": "collector1",
+                        "collector_ip": "10.100.12.13",
```

Comment:
Do we need the test case for IPv6 collector IP?

Reply:
Added a case